---
string model_name
string camera_revision
string serial_number
string firmware_version
